scripts/versions.html.template

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
```

Comment:
@daspecster This will definitely need to change / probably shouldn't exist, since we are using the readthedocs theme now.